### ISSUE TITLE
Support playback of files with .strm file extension

### ISSUE THREAD

**Issue by MVasuki1** [mpv-android/mpv-android#721]
strm files will just have a single stream url
https://emby.media/support/articles/Strm-Files.html

It can be handled similar to how mpv-android handles m3u files

**Comment by sfan5** [comment]
This seems redundant, you can easily use the M3U format in almost identical fashion.
Do these even have a mime type assigned/in-use? Do any other programs support them?

**Comment by MVasuki1** [comment]
Yes, Emby and Jellyfin support extracting metadata for strm files and not m3u, and it allows using mpv as external player.
But since mpv does not recognize strm files. I just decided to recompile jellyfin-server after replacing strm with m3u everywhere in source code (of jellyfin-server).

It would be nice if mpv supports strm files directly, so that I need not have to do this for newer releases of jellyfin

**Comment by sfan5** [comment]
upstream contribution: https://github.com/mpv-player/mpv/pull/17793
